### DISCUSSION TITLE
Google SDE Internship 2024

### DISCUSSION THREAD

**Discussion by ahmetdumlu** [jenndryden/Canadian-Tech-Internships-and-New-Grad-2025#78] [General]
When did it get opened and closed? Isn't it too early to close?

**Comment by Adamose** [comment]
Opened june 6 and closed june 12
they most likely will reopen in august

**Comment by Ksdsk** [comment]
I heard this opening was for recruiter-contacts only. Can anyone confirm?

**Reply by tohmze** [reply]
I believe so
